src/core/settings.h

```diff
@@ -92,13 +93,14 @@ typedef struct QUIC_SETTINGS_INTERNAL {
     uint8_t SendBufferingEnabled            : 1;
     uint8_t PacingEnabled                   : 1;
     uint8_t MigrationEnabled                : 1;
-    uint8_t DatagramReceiveEnabled          : 1;
+    uint8_t DatagramReceiveEnabled          : 1; 
```

Comment:
Did you add a space here? 😄 Please remove

Reply:
![image](https://github.com/microsoft/msquic/assets/43735701/4218906e-eb56-4164-b6d2-ebff6554d19e)